### DISCUSSION TITLE
关于一个小问题的反馈

### DISCUSSION THREAD

**Discussion by ycloong** [tangly1024/NotionNext#309] [General]
# 文章访客问题
当我把阅读统计关掉时，文章顶部的描述中却没有关闭相应的描述，如下图
![image](https://user-images.githubusercontent.com/83064739/173863811-985ce409-e829-4fcd-8f45-3d8bc03d4b7f.png)

附上博客地址：
https://ycloong.top/article/1079bb4b-c137-47b4-a23d-db554700cdea

**Comment by tangly1024** [comment]
已收到反馈，将在后续版本跟进 https://github.com/tangly1024/NotionNext/issues/310
